src/scripts/main.js

```diff
@@ -354,7 +354,29 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard');
 
-// write your code here
+people.forEach((hum) => {
+  hum.age = hum.died - hum.born;
+  hum.century = Math.ceil(hum.died / 100);
+  hum.sex = hum.sex === 'm' ? "Male" : "Female";
```

Comment:
It is important to use the same quotation marks everywhere
```suggestion
  hum.sex = hum.sex === 'm' ? 'Male' : 'Female';
```